rep.go

```diff
@@ -56,6 +57,11 @@ func (rep *repSocket) Type() SocketType {
 	return rep.sck.Type()
 }
 
+// Addr returns the listen address
```

Comment:
please adapt with whatever updated `zmq4.Socket.Addr` documentation you came up with.